core/src/main/java/nl/nn/adapterframework/configuration/IbisContext.java

```diff
@@ -358,8 +358,6 @@ private void digestClassLoaderConfiguration(ClassLoader classLoader, String curr
 					}
 				}
 
-				ConfigurationDigester configurationDigester = configuration.getBean(ConfigurationDigester.class);
-				configurationDigester.digestConfiguration();
```

Comment:
Is dit verhuist naar Configuration.start(), (waarvan ik denk dat het Configuration.load() moet zijn)